Python/optimizer.c

```diff
@@ -1659,4 +1668,61 @@ _Py_Executors_InvalidateAll(PyInterpreterState *interp, int is_invalidation)
     }
 }
 
+void _Py_Executor_Invalidate(_PyExecutorObject *executor)
+{
+   if (executor->vm_data.valid) {
+    unlink_executor(executor);
+    executor_clear(executor);
+   }
+}
```

Comment:
If we're not using this, let's get rid of it. It's not particularly hard to implement if/when we find the need. :)